Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Unisolate_Endpoint6_2.yml

```diff
@@ -0,0 +1,352 @@
+      scriptName: IsIntegrationAvailable
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "no":
+      - "5"
+      "yes":
+      - "4"
+    scriptarguments:
+      brandname:
+        simple: Cortex XDR - IR
+    results:
+    - brandInstances
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 1470,
+          "y": 260
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    continueonerrortype: ""
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: f2ee88e0-4fbb-4b5e-81e6-631b37e066ad
+    type: condition
+    task:
+      id: f2ee88e0-4fbb-4b5e-81e6-631b37e066ad
+      version: -1
+      name: Is there an endpoint to unisolate?
+      description:  Is there an endpoint to unisolate?
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "5"
+      "yes":
+      - "7"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: inputs.Endpoint_ID
+            iscontext: true
+          right:
+            value: {}
+        - operator: isNotEmpty
+          left:
+            value:
+              simple: inputs.Hostname
+            iscontext: true
+        - operator: isNotEmpty
+          left:
+            value:
+              simple: inputs.IP
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": 1690,
+          "y": 430
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    continueonerrortype: ""
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: 1dd236f7-ea7b-447a-8374-f5b9149e6f04
+    type: title
+    task:
+      id: 1dd236f7-ea7b-447a-8374-f5b9149e6f04
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 1470,
+          "y": 1160
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    continueonerrortype: ""
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "6":
+    id: "6"
+    taskid: 7f93febb-d3a1-429a-843a-08b7143d52bd
+    type: regular
+    task:
+      id: 7f93febb-d3a1-429a-843a-08b7143d52bd
+      version: -1
+      name: Unisolate endpoint
+      description: Reverses the isolation of an endpoint.
+      script: '|||xdr-endpoint-unisolate'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      endpoint_id:
+        complex:
+          root: inputs.Endpoint_ID
```

Comment:
let's take the input from the get endpoint status task (like what we talked about with the isolate playbook)